iot-e2e-tests/edge-e2e/src/main/java/glue/ModuleGlue.java

```diff
@@ -161,7 +161,7 @@ public void connect(String transportType, String connectionString, Certificate c
             ConnectResponse cr = new ConnectResponse();
             cr.setConnectionId(connectionId);
             handler.handle(Future.succeededFuture(cr));
-        } catch (ModuleClientException | URISyntaxException | IOException e)
+        } catch (URISyntaxException | IOException e)
```

Comment:
If you're fixing brace styles, want to fix this one too?

Reply:
Sure, I can do that